cicd/multi-gitter-config

```diff
@@ -0,0 +1,39 @@
+base-branch:
+  - main
+branch: from-multi-gitter
+concurrent: 20
+conflict-strategy: skip
+draft: false
+dry-run: false
+fetch-depth: 1
+fork: true
+fork-owner:
+git-type: cmd
+interactive: false
+log-file: "-"
+log-format: text
+log-level: debug
+max-reviewers: 0
+max-team-reviewers: 0
+org:
+  - ZOSOpenTools
+output: "-"
+plain-output: false
+platform: github
+pr-title: "multi-gitter: Add GitHub workflows"
+repo-include: ".*port$"
+skip-forks: false
+skip-pr: false
+skip-repo:
+  - ZOSOpenTools/zotsampleport
```

Comment:
Curious why these are skipped?

Reply:
Languages & compilers - We're not releasing anything. So they don't need bump workflows.
kafka - needs [this](https://github.com/ZOSOpenTools/librdkafkaport/pull/5) merged first
llama - needs [this](https://github.com/ZOSOpenTools/llamacppport/pull/1) merged first